src/main/ingestion-queues/queue.ts

```diff
@@ -51,48 +51,76 @@ export async function startQueue(
     try {
         if (server.KAFKA_ENABLED) {
             return await startQueueKafka(server, piscina, mergedWorkerMethods)
```

Comment:
The `return` doesn't make sense here anymore, does it? I think you want `startQueueRedis` always?